src/go/rpk/pkg/cli/cmd/generate/prometheus.go

```diff
@@ -86,7 +86,7 @@ especify an arbitrary config file.`,
 A comma-delimited list of the addresses (<host:port>) of all the redpanda nodes
 in a cluster. The port must be the one configured for the nodes' admin API
 (%d by default)`,
-			config.Default().Redpanda.AdminApi.Port,
+			config.Default().Redpanda.AdminApi[0].Port,
```

Comment:
When `kafka_api` changed to be a list instead of a single object, I added a const for the default Kafka port, (https://github.com/vectorizedio/redpanda/blob/dev/src/go/rpk/pkg/config/config.go#L30), turning lines like this
```
config.Default().Redpanda.KafkaApi[0].Port
```
into
```
config.DefaultKafkaPort
```
which is much less verbose. I think this change could benefit from taking that approach too.

Reply:
Ah, I just saw you added it [here](https://github.com/vectorizedio/redpanda/pull/1130/commits/625904378eea8011c39d5d453e580dfbd06bc749#diff-9a296ba5e3ab07ab5408e1cfa2b9fea4c3dfbc56ff032d3390f7b44ad0cb9ddeR31), so I think it makes sense to use it here too.

Reply:
Thanks, using `config.DefaultKafkaPort` now